crowbar_framework/spec/models/backup_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe Backup do
   let(:created_at) { Time.zone.now.strftime("%Y%m%d-%H%M%S") }
+  let!(:stub_methods) { [:validate_file_extension, :validate_version, :validate_hostname, :create_archive] }
```

Comment:
Metrics/LineLength: Line is too long. [108/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)